Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1

```diff
@@ -101,12 +102,14 @@ function New-DiffPackage
             }
             Write-Host (Get-VstsLocString -Key DIFFPKG_CreatingDiffPackageForService -ArgumentList @($localServiceManifestName, $clusterServiceManifest.Version, $localServiceManifestVersion))
 
+            $diffServicePkgPath = New-Item -ItemType Directory -Path $diffServicePkgPath -Force
```

Comment:
should we check and ignore if the path is already present

Reply:
Prior to this, we already remove temporary diff folder if it exists. It is better to start with a clean diff directory.